### ISSUE TITLE
Support for relative offsets

### ISSUE THREAD

**Issue by atamon** [Blizzard/node-rdkafka#430]
Hi,

Thanks for maintaining a great kafka library for nodejs! 

#### TL;DR
**Does node-rdkakfa support "relative offsets" In some way?**. That is `offset=-100` should give me the last 100 offsets before any new ones start arriving.

#### Background
One of my node-rdkakfa use-cases is reading a stream of "configuration updates". Upon service start it becomes apparent that we want to read the last message.

This is a common debug use-case we've seen as well. For ad-hoc debugging of kafka-related issues we often use kafkacat (https://github.com/edenhill/kafkacat). Which is a wrapper for librdkafka just as this project.

kafkacat seems to accomplish this through: https://github.com/edenhill/kafkacat/blob/be0cfa615b4e854f8b3b437fd80f53bd72c111a3/kafkacat.c#L1427. And I fail to find a reference to something like this in node-rdkafkas source-tree.

Thanks for your time!
